<?xml version="1.0"?>

<launch>
    <arg name="mode" default="demo" />
    
    <!-- High level formation controller -->
    <node name="formationControl" pkg="formation_control" type="formationControl.py" output="screen">
        <rosparam command="load" file="$(find formation_control)/config/demo_param.yaml"/>
        <param name="mode" value="$(arg mode)" />
        <remap from="cmd_vel" to="des_vel" />
    </node>

    <!-- low level WMR controller -->
    <node name="lowLevelController" pkg="formation_control" type="turtlebotControl.py" output="screen"/>
</launch>
